l2geth/rollup/sync_service.go

```diff
@@ -747,7 +745,7 @@ func (s *SyncService) ApplyTransaction(tx *types.Transaction) error {
 	}
 
 	// Set the raw transaction data in the meta
```

Comment:
Technically the better way to do this is use the raw RLP sent via RPC so that it doesn't need to be reserialized here. The `txmeta` is created in `eth_sendRawTransaction` handler and `rawTransaction` can be set there

Reply:
How would you do this?

Reply:
This is the line that needs updating: https://github.com/ethereum-optimism/optimism/blob/eef1df4d63fd1b483beda63293ea1746861c3e2a/l2geth/internal/ethapi/api.go#L1738

The RLP encoded tx is in scope as a `hexutil.Bytes`

Reply:
When the `RollupClient` fetches a tx, it sets the `rawTransaction` here: https://github.com/ethereum-optimism/optimism/blob/eef1df4d63fd1b483beda63293ea1746861c3e2a/l2geth/rollup/client.go#L329